<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestCenterWindow</class>
 <widget class="QMainWindow" name="TestCenterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>766</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Test Center</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Enter hex numbers separated by &quot;,&quot; &quot;.&quot; or blank</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="box_test_list">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_open_list">
         <property name="text">
          <string>Open list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_save_list">
         <property name="text">
          <string>Save list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_add_to_list">
         <property name="text">
          <string>Add to list</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_send">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_copy">
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string> </string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="check_box_add_check">
         <property name="text">
          <string>Add Check</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_button_crc">
         <property name="text">
          <string>CRC</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_button_lrc">
         <property name="text">
          <string>LRC</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="text_edit_traffic"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
